lib/grpc-lwt/client.ml

```diff
@@ -44,7 +44,7 @@ let call ~service ~rpc ?(scheme = "https") ~handler ~(do_request : do_request)
   match response.status with
   | `OK ->
       let+ status =
-        match H2.Headers.get headers "grpc-status" with
+        match H2.Headers.get response.headers "grpc-status" with
         | Some _ -> Lwt.return (Grpc.Status.extract_status headers)
```

Comment:
```suggestion
        | Some _ -> Lwt.return (Grpc.Status.extract_status response.headers)
```

Reply:
This should also extract the status from the response headers, no?

Reply:
Thanks, I completely overlooked that second instance of `headers`. I have made the change in the latest version of the PR.